src/github.com/getlantern/balancer/balancer.go

```diff
@@ -107,7 +108,7 @@ func (b *Balancer) Dial(network, addr string) (net.Conn, error) {
 		log.Tracef("Dialing %s://%s with %s", network, addr, d.Label)
 		conn, err := d.dial(network, addr)
 		if err != nil {
-			log.Errorf("Unable to dial via %v to %s://%s: %v on pass %v...continuing", d.Label, network, addr, err, i)
+			log.Error(errors.New("Unable to dial via %v to %s://%s: %v on pass %v...continuing", d.Label, network, addr, err, i))
```

Comment:
Yeah, we've built something cool here!  Your error abstraction worked out very nicely.